src/NuGet.Core/NuGet.CommandLine.XPlat/Program.cs

```diff
@@ -43,123 +46,167 @@ public static int Main(string[] args)
             var app = new CommandLineApplication();
             app.Name = "nuget3";
             app.FullName = Strings.App_FullName;
-            app.HelpOption("-h|--help");
+            app.HelpOption(HelpOption);
             app.VersionOption("--version", typeof(Program).GetTypeInfo().Assembly.GetName().Version.ToString());
 
             var verbosity = app.Option("-v|--verbosity <verbosity>", Strings.Switch_Verbosity, CommandOptionType.SingleValue);
```

Comment:
Right, but this doesn't use the constant. Also, I'm not sure it's needed since you added it to the sub-command (but I think we need to keep it in the short term because dotnet-restore expects that it can pass it that way)